docs/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -19,7 +19,15 @@ const sidebarSections = SIDEBAR[langCode].reduce((col, item) => {
 ---
 
 <nav aria-labelledby="grid-left">
-  <ul class="nav-groups">  
+  <ul class="nav-groups">
+    <li>
+      <div class="nav-group">
+        <h2 class="sponsors-title">Sponsored by</h2>
+        <ul class="sponsors">
+          <li><a href="https://www.netlify.com/"><img width="100" height="27" src="/assets/netlify.svg" alt="Netlify" /></a></li>
```

Comment:
this might be the biggest nit of my life, but here goes: "104" brings it totally aligned with the "sponsored by" text on my screen 🤦

Reply:
omg 😂 just for that I will do it haha